components/profile/AccountSettings.tsx

```diff
@@ -0,0 +1,174 @@
+      {
+        headers: {
+          "Content-Type": "application/json",
+          Authorization: `Token ${currentUser?.token}`,
+        },
+      }
+    );
+
+    if (status !== 200) {
+      setErrors(data.errors.body);
+    }
+
+    if (data?.user) {
+      window.localStorage.setItem("user", JSON.stringify(data.user));
+      mutate("user", data.user);
+    }
+  };
+  const Reupload = async (value) =>{
+    const files= value.target.files
+    const data_img = new FormData();
+    data_img.append("file",files[0]);
+      data_img.append("upload_preset", 'upload')
+      const res = await fetch("https://api.cloudinary.com/v1_1/rajshah/upload",{
+          method:'POST',
+          body: data_img
+    });
+    const response = await res.json();
+    const state = userInfo;
+    const newState = { ...state, 'image': response.secure_url };
+    const user = { ...newState };
+    if (!user.password) {
+      delete user.password;
+    }
+
+    const { data, status } = await axios.put(
+      `${SERVER_BASE_URL}/user`,
+      JSON.stringify({ user }),
+      {
+        headers: {
+          "Content-Type": "application/json",
+          Authorization: `Token ${currentUser?.token}`,
+        },
+      }
+    );
+
+    if (status !== 200) {
+      setErrors(data.errors.body);
+    }
+
+    if (data?.user) {
+      window.localStorage.setItem("user", JSON.stringify(data.user));
+      mutate("user", data.user);
+    }
+    setUserInfo(newState);
+  }
+
+  return (
+    <React.Fragment>
+          <Row>
+          <Col span={12}>
+              <Col span={24}>
+                <h6>User Profile</h6>
+                <Avatar src= {userInfo.image} size = {50}/>
+                <br/>
+                <br/>
+                <label style={{color:"black"}}>Reupload Image<input style={{display:"none"}} type="file" onChange={Reupload}/></label>
+              </Col>
+              <br/>
+              <Col span={24}>
+              <h6>Github</h6>
+              <Text style={{background:"white",padding:"2%"}}>{userInfo.github || "www.github.com"}</Text><Text style={textStyle}>verified</Text>
+              </Col>
+              <br/>
+              <Col span={24}>
+              <h6>Twitter</h6>
+              <Text style={{background:"white",padding:"2%"}}>{userInfo.twitter || "www.twitter.com"}</Text><Text style={textStyle}>verified</Text>
+              </Col>
+              <br/>
+              <Col span={24}>
+              <h6>LinkedIn</h6>
+              <Text style={{background:"white",padding:"2%"}}>{userInfo.linkedIn || "www.linkedin.com"}</Text><Text style={textStyle}>verified</Text>
+              </Col>
+              <br/>
+              <Col span={24}>
+              <h6>Personal Website</h6>
+              <Input type="text" style={{background:"white",padding:"2%",width:"50%"}} onChange={updateState("website")} defaultValue={userInfo.website} placeholder={"www.example.com"}/><button style={textStyle} onClick={submitForm}>edit</button>
```

Comment:
There should be a white border instead of a gray one. The input bar is too fat. Like the other comment, use ant design's input component, https://ant.design/components/input/

Reply:
made styling changes